tests/common/common.py

```diff
@@ -128,9 +129,19 @@ def trigger_and_retrieve(
             try:
                 log_list = self._get_logs(filter_str, ignore_protos)
                 return log_list
-            except (LogsNotFound, RpcError) as e:
+            except RpcError as e:
+                print(f"RPC error: {e}")
+                # most RpcErrors come from exceeding the reads per minute quota
+                # wait at least 60 seconds
+                # use a randomized backoff so parallel runs don't start up at 
+                # the same time again
+                sleep(random.randint(60, 300))
+                tries += 1
+            except LogsNotFound as e:
                 print("logs not found...")
-                sleep(5)
+                # logs may not have been fully ingested into Cloud Logging
+                # Wait before trying again
+                sleep(10 * (tries+1))
                 tries += 1
```

Comment:
nit: would it theoretically lead to an endless loop if logs never found? i would propose to add a threshold.